1 table.sql

```diff
@@ -0,0 +1,68 @@
+use FlyAirDB
+go 
+drop table if exists dbo.FlyAir
+go 
+
+create table dbo.FlyAir (
+    FlyAirID            int not null identity primary key,
+    FlightNumber        varchar (6) not null        constraint ck_Flight_number_must_start_with_FLY_followed_by_three_numbers check (FlightNumber like 'FLY[0-9][0-9][0-9]'),
+    DepartureAirport    varchar (4) not null        constraint ck_Departure_Airport_must_be_between_3_and_4_letters check (len(DepartureAirport) between 3 and 4 and DepartureAirport not like '%[^A-Z]%'),
+    DepartureCountry    varchar (25) not null       constraint ck_Departure_Country_cannot_be_blank check (DepartureCountry <> ''),
+                                                    constraint ck_Departure_Country_may_only_contain_letters check(DepartureCountry not like '%[^A-Z]%'),
+    ArrivalAirport      varchar (4) not null        constraint ck_Arrival_airport_must_be_between_3_and_4_letters check (len(ArrivalAirport) between 3 and 4 and ArrivalAirport not like '%[^A-Z]%'),
+    ArrivalCountry      varchar (25) not null       constraint ck_Arrival_Country_cannot_be_blank check (ArrivalCountry <> ''),
+                                                    constraint ck_Arrival_Country_may_only_contain_letters check(ArrivalCountry not like '%[^A-Z]%'),
+    DateDeparting       date not null,
+    TimeDeparting       time not null               constraint ck_Time_Departing_cannot_be_blank check (TimeDeparting <> ''),
+    DateArriving        date not null,
+    TimeArriving        time not null               constraint ck_Time_Arriving_cannot_be_blank check (TimeArriving <> ''),
+    PassengerFirstName  varchar (30) not null       constraint ck_Passenger_first_name_cannot_be_blank check (PassengerFirstName <>''),
+    PassengerLastName   varchar (30) not null       constraint ck_Passenger_last_name_cannot_be_blank check (PassengerLastName <>''),
+    DOB                 date not null,
+    PassengerAdress     varchar (50) not null       constraint ck_Passenger_Adress_cannot_be_blank check (PassengerAdress <>''),   
+    PassportNumber      varchar (9)                 constraint ck_Passport_Number_must_caontain_exact_9_digits_not_strating_with_0 check (len(PassportNumber)= 9 and PassportNumber like '[1-9]%' ),
```

Comment:
1. The constraint name has a typo (caontain and strating).
2. The column type should be char(9) instead of varchar(9) since a passport number is always exactly 9 digits.

Reply:
1. Fixed it
2. fFxed it and deleted the  (len)part of the constraint that will not be relevant anymore.
Btw, I did on purpose a varchar, because the error message by a char would just look like this 
"String or binary data would be truncated." compared to a varchar that will return the message from the constraint

Reply:
You can also set the `len` constraint in order to get a nicer message.